### ISSUE TITLE
对于不同 CS ratio /噪声 /测量条件，是否有推荐 hyper-parameter 配置？

### ISSUE THREAD

**Issue by 1761790781-sketch** [ICSResearch/NesTD-Net#6]
compressive sensing 的测量率 (CS ratio) 与噪声情况会显著影响重建效果，不同设置下可能需要不同的超参数 (learning rate, epochs, warmup, regularization 等)。
目前 README 只给出一个训练示例 (cs_ratio=10，image_size=132)，没有给出针对其他常见情形 (更低测量率 /不同数据集) 的建议。
建议作者补充一些经验参考，或支持 config 文件 / hyper-parameter 列表，方便用户复现实验或迁移到新任务。
